instrumentation/opentelemetry-instrumentation-psycopg/src/opentelemetry/instrumentation/psycopg/__init__.py

```diff
@@ -269,7 +269,8 @@ def get_operation_name(self, cursor, args):
         if isinstance(statement, Composed):
             statement = statement.as_string(cursor)
 
-        if isinstance(statement, str):
+        # `statement` can be empty string. See #2643
```

Comment:
Instead of the comment, would be nice to add a testcase so that we are sure we don't regress